src/layout/drawablemark.cpp

```diff
@@ -58,7 +58,7 @@ CADrawableMark::CADrawableMark(CAMark* mark, CADrawableContext* dContext, double
         font.setPixelSize(qRound(DEFAULT_TEXT_SIZE));
         QFontMetrics fm(font);
 
-        int textWidth = fm.width(static_cast<CAText*>(this->mark())->text());
+        int textWidth = fm.horizontalAdvance(static_cast<CAText*>(this->mark())->text());
```

Comment:
`horizontalAdvance()` was introduced in Qt 5.11, so please also wrap this one inside
```
#if QT_VERSION >= ...
```

Reply:
OK, will wrap those 7 calls in QT_VERION check.